### ISSUE TITLE
Extract syllabus for courses and add support to requests for the same.

### ISSUE THREAD

**Issue by Ayushk4** [kshitij10496/hercules#43]
This can be sent as just another key in the endpoint for course -timetable ( see https://hercules-10496.herokuapp.com/api/v1/course/timetable/MA61023) or a separate endpoint. We can use this thread to discuss regarding the same.

As for how to extract syllabus, one may refer link :
( https://erp.iitkgp.ac.in/Acad/subject/get_syllabus_pdf.jsp?subno= <course_code_here> &year=20181 )

**Comment by kshitij10496** [comment]
We can create a new endpoint: `/course/info/{course_code}` for fetching all the related information to a course such as the faculty, credits and syllabus.

**Comment by Ayushk4** [comment]
Yes the thing that I am building would need the faculty - name that's offering the course.
